hack/environment.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+get_cd_registry () {
+  # for creating and uploading a component descriptor just insert the location to your registry e.g. eu.gcr.io/.../cnudie/gardener/development
+  registry=""
```

Comment:
I think you should remove this line. In this way I can set this environment variable with the value that I desire. Otherwise this `get_cd_registry` will always fail.